test/unit/pages/confirm/controller-test.js

```diff
@@ -104,6 +105,29 @@ describe('ConfirmController', function() {
     });
   });
 
+  describe('isConfigurationValid', function() {
+    it('should able to check JBDS is configured', function() {
+      let stub = sandbox.stub(ConfirmController.prototype, 'jbdsIsConfigured').returns(true);
+      confirmController.isConfigurationValid();
+      expect(stub).calledOnce;
+      expect(confirmController.jbdsIsConfigured()).to.be.true;
```

Comment:
Then you check if stabbed method returns true. What are you testing? It looks like you testing if sinon stubbing is working correctly :) Instead you should stub all three involved methods and that would be white box testing.  Se my comment above and try to implement test with second approach.